packages/clay-alert/package.json

```diff
@@ -4,18 +4,21 @@
 	"description": "ClayAlert component",
 	"license": "BSD-3-Clause",
 	"repository": "https://github.com/liferay/clay",
-	"engines": {
-		"node": ">=0.12.0",
-		"npm": ">=3.0.0"
+	"main": "lib/cjs/index.js",
+	"module": "lib/esm/index.js",
+	"exports": {
+		"import": "./lib/esm/index.js",
+		"require": "./lib/cjs/index.js"
```

Comment:
These are all standards with package.json now and allows for both ESM and CJS consumption